Source/src_android/PinCheckerLibrary/src/main/java/com/wultra/android/passphrasemeter/PasswordTester.java

```diff
@@ -26,6 +26,9 @@
 
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
```

Comment:
One last thing, please hit "optimize imports" on the whole project. There are few unused imports forgot by you and by us.